### DISCUSSION TITLE
Disable persisting batch metadata

### DISCUSSION THREAD

**Discussion by nikola-golijanin** [spring-projects/spring-batch#4639] [Q&A]
Is there a simple way to disable job metadata persistence, in case we just need to read -> process -> write.

**Comment by fmbenhassine** [comment]
Starting from v5, the map-based job repository was removed. If you don't care about meta-data, you can use the jdbc-based job repository with an in-memory database like h2 or hsqldb.

Have you tried that? This is the recommended replacement for the removed map-based repository.
